Packs/ipinfo/Integrations/ipinfo_v2/ipinfo_v2.py

```diff
@@ -0,0 +1,197 @@
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Any
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+class Client(BaseClient):
+    def __init__(self, api_key: str, base_url: str, verify_certificate: bool, proxy: bool,
+                 reliability: DBotScoreReliability):
+        """
+        Client to use in the IPinfo integration. Uses BaseClient
+        """
+        super().__init__(base_url=base_url, proxy=proxy, verify=verify_certificate)
+        self.api_key = api_key
+        self.reliability = reliability
+
+    def _url_suffix(self, ip: str) -> str:
+        api_key_in_url = f'?token={self.api_key}' if self.api_key else ''
+        return f'{ip}/json{api_key_in_url}'
+
+    def ipinfo_ip(self, ip: str) -> Dict[str, Any]:
+        return self._http_request(method='GET', url_suffix=self._url_suffix(ip=ip))
+
+
+def test_module(client: Client) -> str:
+    """Tests IPinfo by sending a query on 8.8.8.8"""
+    client.ipinfo_ip('8.8.8.8')
+    return 'ok'  # on any failure, an exception is raised
+
+
+#
+
+def ipinfo_ip_command(client: Client, ip: str) -> List[CommandResults]:
+    response = client.ipinfo_ip(ip)
+    return parse_results(ip, response, client.reliability)
+
+
+def parse_results(ip: str, raw_result: Dict[str, Any], reliability: DBotScoreReliability) -> List[CommandResults]:
+    command_results = []
+
+    # default values
+    asn = as_owner = None
+    feed_related_indicators = []
+
+    if raw_result:
```

Comment:
This is a personal taste, but since the rest of the function is indented, maybe make this check a negative - i.e. `if not raw_result: return command_results`.
Having a single `return` is a nice practice, but I personally feel the code is easier to read without function long indentations.